scan.go

```diff
@@ -5,13 +5,13 @@
 package elastic
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
+	"gopkg.in/olivere/elastic.v3/uritemplates"
```

Comment:
Please keep the import statements intact.

1. stdlib imports
2. Non-stdlib imports
3. Application imports